internal/suggestions/suggestions.go

```diff
@@ -237,7 +237,7 @@ func (s *Suggestions) findSuggestion(validationErr error, previousSuggestionCont
 func (s *Suggestions) getSuggestionAndRevalidate(validationErr error, previousSuggestionContext *string) (*Suggestion, []byte, error) {
 	suggestion, err := s.findSuggestion(validationErr, previousSuggestionContext)
 	if err != nil {
-		return nil, nil, err
+		return nil, nil, fmt.Errorf("%w: %v", ErrNoSuggestionFound, err)
```

Comment:
wrap `ErrNoSuggestionFound`. This allows us to continue to next validation issue when we can't get a suggestion for the current one. Helpful when we tell the LLM to "give up".

I sometimes run in serial and am considering changing the action to do so as well. The output is a lot more understandable + predictable (at least until we figure out pre-processing). There are issues that happen quite often without that and the speedup is pretty negligible currently, so I think serial can make it much easier to show customers good suggestions atm.